### ISSUE TITLE
Full Revision of Staggered Orders

### ISSUE THREAD

**Issue by MarkoPaasila** [Codaone/DEXBot#237]
Work in progress. This will replace all other _Staggered Orders_-related issues. I will update this description at some point.

**Comment by lsji07** [comment]
Detection of exceeded strategy range logic could be enhanced. To my mind this should move bounds when the spot or centre price moves significantly.

One way to do it is if an order trade is detected within a configurable percent of any bound the strategy range should increase according to the set strategy.

**Comment by MarkoPaasila** [comment]
Yes good idea. Maybe an input field with a default of +-5x, so unless the user explicitly decreases it, it is a relatively safe value. I won't add this to the logic just yet - to avoid feature creep, but add this feature to the queue.
